playbackScreen/src/main/kotlin/voice/playbackScreen/BookPlayViewModel.kt

```diff
@@ -182,9 +187,32 @@ class BookPlayViewModel
         }
       }
     }
+    autoSleepTimerPref.data.map { prefs ->
```

Comment:
This won't work like this.
Calling `map` transforms the flow, but without calling `collect()` at some point, the code inside map will never be executed.

Instead you usually use something like:
```kotlin
scope.launch {
  flow.collect { prefs->
    // logic here
  }
}
```